spec.bs

```diff
@@ -3825,6 +3865,23 @@ from querying the server during an auction.
     1. Return true if it is above the threshold, otherwise return false.
 </div>
 
+<div algorithm>

```

Comment:
Could you please move this new algorithm down to just before or after `compute the key hash of reporting ID`, since that's the only place it's used, so that the association between the two is more obvious?